js/index.js

```diff
@@ -1,7 +1,88 @@
+
+/*let wordCount = 0;
+  for (let i = 0; i < text.length; i++) {
+	if (text[i] != " ") {
+		wordCount += 1;
+  	}
+  }
+
+console.log(wordCount);*/
+
+/* sin el metodo .split me cuenta cada caracter de la función en lugar de cada palabra...no logro entender como puedo sortear el problema. He considerado que un 'string' es como unos caracteres sin espacio y he creado una condicion según la cual a cada iteración, sí el index del text es diferente al espacio en blanco (" ") me suma 1 a la variable wordCount...pero no...Con el método .split lo que hago, al final, es hacer que la variable text se "separa" en un array de cadenas y contar cuanto es largo el array resultante*/
+
+//console.log(wordCount.split(" ").length);
+
+/*Tampoco logro la segunda parte del 1er bonus...Separo nuestra variable en un array de cadenas y lo guardo en una nueva variable
+
+let textArr = text.split(" ");
+
+//busco el patrón en este caso 'et'
+let charCount = 0;
+
+for (let i = 0; i < textArr.length; i++) {
+  if (textArr[i] === 'et') {
+    charCount++;
+  }
+}
+
+console.log(charCount);
+
+//el resultado es 3 pero debería ser 5...*/
```

Comment:
You did it right!
I just see 4 "et" in the "text" string. The result here is only 3, because the last one is actually "et," (with a comma), so you could for this case in the condition.
`if (textArr[i] === 'et' || textArr[i] === 'et,') {
    charCount++;
  }`